component/section/section.css

```diff
@@ -0,0 +1,202 @@
+.section {
+    box-sizing: border-box;
+    width: 100%;
+    display: inline-flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    font-family: Verdana, Geneva, Tahoma, sans-serif;
+    color: #232323;
+}
+
+.section__header {
+    font-weight: bold;
+    color: #787d94;
+    width: 100%;
+    padding-bottom: 20px;
+    text-align: center;
+    min-width: 300px;
+}
+
+.section__header hr {
+    width: 200px;
+    height:2px;
+    border: 0;
+    background: #787d94;
+}
+
+.section__bio {
+    padding-top: 10px;
+    align-items: center;
+    flex-direction: column;
+    justify-content: center;
+    background-image: url('../../img/bioBg.png');
+    background-repeat: no-repeat;
+    background-size: 100% 100%;
+    /* background-position-y: -700px; */
+    color: #77df00;
+    height: 400px;
+}
+
+.section__bio h2 {
+    margin-top: -10px;
+    color: #b4a080;
+}
+
+.section__welcome {
+    display: flex;
+    align-items: center;
+    background-color: #ffffff;
+    color: #232323;
+    text-align: justify;
+    justify-content: center;
+    margin-bottom: 20px;
+}
+
+.section__welcome p {
```

Comment:
With these "section__welcome", "section__about", and such, you're being as specific as an id without using an id. This should really be a modifier on "section". Try to think of the "style" difference when naming class modifiers too. If your styling is really that different between sections (as far as things like font size) then you probably also should be standardizing those too anyways.

Reply:
Oh, i see. I can combine all the similarities on "section", and then only add the individual differences on each separate one like "section_welcome" and so on. Will have some refactoring to do :) . But this just clicked for me. Thanks for the comment!